### ISSUE TITLE
Settings dialog, TTS voices page: presentation of preferred voices

### ISSUE THREAD

**Issue by marisademeglio** [daisy/pipeline-ui#230]
- The list of preferred voices basically represents the voice config XML. In that regard, I thought it might be a good idea to present it in such a way that it makes it more obvious how the voice selection will work. This is currently left up to the user to guess, and I think it's pretty intuitive so should be fine, but still... The user might not realize at first that an American English preferred voice will be used for any English content (unless another preferred voice is selected as default). So perhaps each preferred voice could have a column or ℹ button that tells you in plain English when the voice will be used, similar to how I did it [in my comment above](https://github.com/daisy/pipeline-ui/issues/170#issuecomment-2142301299).

One final thing: I noticed that while searching for a voice I'm kind of missing seeing the filtered voice list/table instantly, like we had previously. E.g. you used to be able to see all the English voices with corresponding accent and gender, without having to decide on an accent and gender first. That is not possible anymore. But I guess that is the sacrifice for having a simpler and more accessible interface.

_Originally posted by @bertfrees in https://github.com/daisy/pipeline-ui/issues/170#issuecomment-2147207769_

**Comment by marisademeglio** [comment]
Leaving a note here pointing to #184 which discusses how bad the screen reader experience is for a giant voices table. I don't think it's out of the question to show many voices at once but just so we track the challenges with that.

**Comment by prashantverma2014** [comment]
I now see text like "The default voice for Hindi is आरती" at the bottom of table in Preferred voices tab. This is very helpful.

**Comment by marisademeglio** [comment]
Closing this issue, it sounds like the preferred voices presentation is navigable and makes sense.

Also noting that the browse voices table (mentioned above) is back and paginated so as to be smaller/more manageable.
